js/runtime.js

```diff
@@ -1050,6 +1070,12 @@
             randomPoint: function randomPoint(){
                 return new util.Vector(util.randInt(Event.stage.width), util.randInt(Event.stage.height));
             },
+            unitVector: function unitVector(){
```

Comment:
Some conveniences. Unit vector is always <1,1> and zero vector is always <0,0>. Zero vector is good for initializing shapes to use as sprites.